CHANGELOG.md

```diff
@@ -3,6 +3,62 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v2.1.0 - [date]
```

Comment:
Update date when releasing!

Reply:
I also missed in mine.....how do you do this? just direct commit to dev ?